dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-webexteams/src/main/java/org/apache/dolphinscheduler/plugin/alert/webexteams/WebexTeamsSender.java

```diff
@@ -67,16 +68,28 @@ public AlertResult sendWebexTeamsAlter(AlertData alertData) {
         alertResult.setStatus("false");
         alertResult.setMessage("send webex teams alert fail.");
 
-        try {
-            send(alertResult, alertData);
-        } catch (Exception e) {
-            log.info("send webex teams alert exception : {}", e.getMessage());
+        for (int i = 0; i < AlertConstants.RETRY_TIMES; i++) {
```

Comment:
I'm voting -1. There is a bad code smell here, with some duplicate code.

Reply:
Yes, some good suggestions about this feature? Different plugin maybe have some same logic, but not all plugins has this logic

Reply:
In fact, many Alert plugins construct an HTTP request and send it to the destination.  The differences lie only in the construction of HTTP headers, HTTP body, HTTP method, and HTTP response parsing.  Would it be possible to establish an HTTP client with a built-in retry function, which all Alert plugins could use?

Reply:
Good question, I will update this logic